src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginCache.cs

```diff
@@ -117,8 +117,8 @@ public List<string> GetExtensionPaths(string endsWithPattern, bool skipDefaultEx
     /// <returns>
     /// The <see cref="Dictionary"/>. of test plugin info.
     /// </returns>
-    public Dictionary<string, TPluginInfo> DiscoverTestExtensions<TPluginInfo, TExtension>(string endsWithPattern)
-        where TPluginInfo : TestPluginInformation
+    public Dictionary<string, TPluginInfo> DiscoverTestExtensions<TPluginInfo, TExtension>(
+        string endsWithPattern) where TPluginInfo : TestPluginInformation
```

Comment:
Nit: Feel free to ignore but as you split parameters to new line, I'd be nice to also split the type constraint.

```suggestion
        string endsWithPattern)
        where TPluginInfo : TestPluginInformation
```

Reply:
Fixed.